.gitignore

```diff
@@ -3,3 +3,5 @@
 /telegraf.exe
 /telegraf.gz
 /vendor
+*.swp
```

Comment:
Revert this file, editor and IDE files belong in your global `.gitignore`. If you don't have one, you can add it with:

```
git config --global core.excludesfile $HOME/.gitignore
```

Then add these entries to `$HOME/.gitignore`